channels.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config({silent: true});
+
+const ds = require('./datastore.js');
+const core = require('./core.js');
+
+var channelStore = new ds.DataStore();
+setInterval(getChannels, 10000);
+
+
+function getChannels () {
+    var options = {
+        url: 'slack.com/api/channels.list?token=' + process.env.SLACK_API_TOKEN
+    };
+
+    core.makeRequest(options, function(data) {
+        var channels = data.channels;
+
+        channels.forEach(function(channel) {
+            channelStore.store([channel.id, 'name', channel.name]);
+        });
+    });
+
+    var openDMOptions = {
+        url: 'slack.com/api/users.list?token=' + process.env.SLACK_API_TOKEN
+    };
+}
```

Comment:
I'd call this something like `refreshChannels` or `fetchChannels` ("get" implies a getter-method to me) and I'd consider not exporting it.